### PR TITLE
[16.0][MIG] pos_global_discount_in_line: Migration to 16.0

### PR DESCRIPTION
**Pull request by FrancoMaxime**

### CODE REVIEW

File: pos_customer_comment/tests/tours/PosCustomerComment.tour.js [hunk 1]
```diff
@@ -4,7 +4,7 @@
     License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
 */
 odoo.define("pos_customer_comment.tour.PosCustomerComment", function (require) {
-    "use strict";
+
```
Comment (legalsylvain) [inline pos_customer_comment/tests/tours/PosCustomerComment.tour.js]: why this change ?

Comment (FrancoMaxime) [reply pos_customer_comment/tests/tours/PosCustomerComment.tour.js]: This is a mistake. II fix it right away

**Comment by baimont** [comment]
functional test is not ok in my opinion:

in v14, seeting a discount percentage with the Add Discount in Line config gives

image

in v16, it adds a new discount line as would the standard do

image

**Comment by FrancoMaxime** [comment]
@baimont Fixed now

**Comment by sanrav** [comment]
Functional testing OK

**Review by sanrav — APPROVED** [review]
Functional testing OK

**Review by apineux — APPROVED** [review]

**Review by ivantodorovich — APPROVED** [review]
Technically LG
